src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexMatch.cs

```diff
@@ -63,8 +63,7 @@ public static Match Empty
         }
 
         internal Match(Regex regex, int capcount, String text, int begpos, int len, int startpos)
-
-        : base(text, new int[2], 0)
+            : base(text, new int[2], 0, "0")
```

Comment:
No, for the unspecified names, we return the index of the numbered group in the regex, as in 0,1,2. Here this Match constructor is used for initilaizing the objects, the base Group constructor, is passing 0 (number of captures), new int[2] for the start and len of the default capture, text = the whole text, hence passing "0" the default group number for the whole successful capture.